EveripediaNetwork/ep-ui
# Revision Feedback section

# Revision Feedback section 

_This PR fixes the revision feedback section not fully visible to the user_

## How should this be tested?

1. Go to the wiki history page of any edited wiki and check any of its previous edits e.g https://iq.wiki/revision/8e21222d-dcd6-4627-9536-89e50579f8ca/
3. Confirm the feedback section is now fully visible on this edited wiki: https://iq-wiki-git-revision-fixes-prediqt.vercel.app/revision/87a21146-3fcd-4350-aa1d-0102e45dbc35/

<details>
<summary> Before </summary>

<img width="1512" alt="Screenshot 2024-09-03 at 16 04 29" src="https://github.com/user-attachments/assets/e0b142e2-3efc-4e90-ab29-3de1f19bbe3c">
</details>

<details>
<summary> After </summary>
<img width="1512" alt="Screenshot 2024-09-03 at 15 59 27" src="https://github.com/user-attachments/assets/e843ff2f-ed59-41bc-be96-46c11308b4ad">

</details>


## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3091
